babel.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  presets: [
+    "@babel/react",
+    "@babel/env",
```

Comment:
why do you need this, Did you specify babel jest in your config?  https://github.com/benfluleck/student-list-full-stack/blob/develop/package.json#L37